Services/Contact/BuddySystem/classes/tables/class.ilBuddySystemRelationsTableGUI.php

```diff
@@ -13,8 +13,7 @@
  * us at:
  * https://www.ilias.de
  * https://github.com/ILIAS-eLearning
- *
```

Comment:
IMO this change is wrong, according to `\ILIAS\CI\Rector\ChangeLicenseHeader`

Reply:
Hmmm, the used header looks identical in `CI/Copyright-Checker/copyright-checker.sh` and `\ILIAS\CI\Rector\ChangeLicenseHeader`. If this header is considered invalid, it is a bug :-).

Reply:
I'll look into it

Reply:
FYI @mjansenDatabay the copyright-checker will only test if the content is similar, the `*`, `/` and space characters will be removed before the comparison. It looks like that was intended =).